src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -1148,4 +1172,11 @@ export default class DefaultAudioVideoController
       );
     });
   }
+
+  private useUpdateTransceiverController(): boolean {
+    return (
+      this.meetingSessionContext.browserBehavior.requiresUnifiedPlan() &&
```

Comment:
That you can't use this with Plan B is useful information to expose in the docs for the policy! It means that N-Scale doesn't scale resolution in some Safari versions, right?

Reply:
I believe unified plan is for all browsers now. Plan-B is originally for Chrome but I think we plan to deprecate it soon. @hensmi-amazon Is that correct?

Reply:
Moreso transition builders off/remove option to turn on before Chrome removes it completely :P, but yes we have an internal task.